WC_Layers/Photon_TallytoVtk.py

```diff
@@ -0,0 +1,35 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Thu Oct  3 23:18:20 2024
+
+@author: Anupama Rajendra
+"""
+import openmc
+import matplotlib.pyplot as plt
+
+def read_statepoint(sp_filename, flux_spectrum_tally_id, mesh_number, summed_axes_energy_filter, energy_filter_index):

```

Comment:
With the addiction of a docstring here, I think some better variable names may emerge:
* `flux_spectrum` => `flux_spectrum_tally`
* `tally_data_reshaped` => `flux_spectrum_mean`
* `flux_sum_energy` => `total_flux`